internal/tfengine/tfengine.go

```diff
@@ -336,3 +338,34 @@ func findUnmanaged(generatedDir, outputDir string, deleteFiles bool) error {
 
 	return nil
 }
+
+// trimDescriptions remove tabs and extra space from schema descriptions
+// so they can be used during rendering without unexpected gaps between lines
+func trimDescriptions(schema map[string]interface{}) {
+	val := reflect.ValueOf(schema)
+
+	for _, key := range val.MapKeys() {
+		v := val.MapIndex(key)
+
+		switch t := v.Interface().(type) {
+		case map[string]interface{}:
+			trimDescriptions(t)
+		case string:
+			if key.String() == "description" {
+				description := v.Interface().(string)
+				trimmedDescription := strings.TrimSpace(leftTrim(description))
+				schema["description"] = trimmedDescription
```

Comment:
Is this the correct map access? Shouldn't it be something like `v[key] = trimmedDescription`?

Reply:
The problem is that v is a reflection, which is not the actual schema map. I'm not too familiar with the concept of reflections and I had trouble writing on it. The alternative would be to add the `struct types` for `schema` and `properties` so we can parse it and traverse more accurately, but I considered an overengineered solution.

Let me know what you think

Reply:
I was more asking if this is correct, I don't think you need to switch to struct types. It just doesn't seem obvious to me, but does it work?

Reply:
Sure, it works.

I decided to send #965 and #964 to show examples of how it shows up.

My only concern would be for deep schema descriptions. The only nested property I had used is [this one](**url**)

Not sure if I can wait to use it to test on deep descriptions such as in `cicd` recipe. But I guess it's fine.

Reply:
I think the "this one" link is broken but I see. LGTM then

Reply:
You're right. My bad. This is the link if you want to take a look:

https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/964/files#diff-e73c9c3e385a38ef46a5b55bef489f61a47403acd9fcfd49728325209d48c2deR80

I'm resolving